pkg/providers/launchtemplate/nodetemplate_test.go

```diff
@@ -707,7 +708,10 @@ var _ = Describe("NodeTemplate/LaunchTemplates", func() {
 		It("should calculate memory overhead based on eni limited pods when ENI limited", func() {
 			ctx = settings.ToContext(ctx, test.Settings(test.SettingOptions{
```

Comment:
Should this be set to false here? If so, do we need a TODO to note that we should drop this when it's no longer supported?

Reply:
Do these tests need to be modified to match v1beta1 behavior or will v1beta1 always behave as though this option is set to true? If it's the latter, I think we keep the test while we still support the configmap and drop the tests later.